### PR TITLE
Added skeleton simulated tests for remainder of plays

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description

This PR adds barebones (ie. no assertions to validation) tests for each remaining play that didn't have one.

I made a few very minor tweaks to the plays themselves, but only for issues that caused AI to crash (degenerate polygons). Otherwise behavior has not changed.

### Testing Done

Existing tests pass, but these new tests don't really do anything.

### Length Justification

Thought it was faster to do all at once.

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by kelvinkoon — DISMISSED** [review]
LGTM, just wanted to ask if there was a specific reason tests were set to 9.5 seconds.

**Comment by MathewMacDougall** [comment]
Uh no just copy pasted from previous tests. The tests timeout at 10 seconds so this way they "pass" the test.

File: src/software/ai/hl/stp/play/free_kick_play.cpp [hunk 1]
```diff
@@ -74,12 +74,12 @@ void FreeKickPlay::getNextTactics(TacticCoroutine::push_type &yield, const World
     if (world.ball().position().x() > -1)
     {
         double y_offset =
-            -std::copysign(world.field().xLength() / 2, world.ball().position().y());
+            -std::copysign(world.field().yLength() / 2, world.ball().position().y());
```
Comment (jonathanlew) [inline src/software/ai/hl/stp/play/free_kick_play.cpp]: What does y_offset mean? Can we rename to something more meaningful?

File: src/software/ai/hl/stp/play/shoot_or_chip_play.cpp [hunk 2]
```diff
@@ -149,12 +132,6 @@ void ShootOrChipPlay::getNextTactics(TacticCoroutine::push_type &yield,
         // We want this second in priority only to the goalie
         result.insert(result.begin() + 1, shoot_or_chip_tactic);
 
-        // If we can't do anything else then patrol?
-        for (auto &patrol_tactic : patrol_tactics)
-        {
```
Comment (jonathanlew) [inline src/software/ai/hl/stp/play/shoot_or_chip_play.cpp:154]: Was this causing a bug?

**Review by jonathanlew — CHANGES_REQUESTED** [review]

File: src/software/ai/hl/stp/play/shoot_or_chip_play.cpp (hunk 2 above)
Comment (MathewMacDougall) [reply src/software/ai/hl/stp/play/shoot_or_chip_play.cpp:154]: It was causing 8 tactics to be assigned so the tactic-assignment code wasn't happy (although it didn't _creash_, just spammed logs)

File: src/software/ai/hl/stp/play/free_kick_play.cpp (hunk 1 above)
Comment (MathewMacDougall) [reply src/software/ai/hl/stp/play/free_kick_play.cpp]: renamed, lmk if that's better.

**Review by jonathanlew — APPROVED** [review]

File: src/software/ai/hl/stp/play/shoot_or_chip_play.cpp (hunk 2 above)
Comment (jonathanlew) [reply src/software/ai/hl/stp/play/shoot_or_chip_play.cpp:154]: adding the patrol tactics was meant to fix a bug where up to 2 of the tactics weren't applicable, but that might not be a problem anymore

**Review by EvanMorcom — APPROVED** [review]
LGTM 

👍

**Review by LiCody — APPROVED** [review]

**Pull request merged**